scipy/stats/_continuous_distns.py

```diff
@@ -6505,41 +6504,42 @@ def scale_mle(loc, data):
             # and Peacock (2000), Page 175
             return (np.sum((data - loc) ** 2) / (2 * len(data))) ** .5
 
+        def loc_mle(loc, data):
```

Comment:
Can we provide any reference here? As a reviewer, I feel like I should check them for accuracy since they're relatively simple.

Reply:
The only reference I have for the two parameter Rayleigh distribution is the notes that I wrote myself, linked above.  To derive the formulas, take the partial derivative of the log-likelihood function with respect to each parameter, and set the result equal to 0. After that, do some algebra to simplify the equations, if possible.

Reply:
I confirmed the equations with the assistance of Sympy.